ee/session_recordings/test/test_session_recording_playlist.py

```diff
@@ -79,6 +79,8 @@ def test_creates_playlist(self):
 
     def test_creates_too_many_playlists(self):
         limit = 0
+        self.organization.available_product_features = AVAILABLE_PRODUCT_FEATURES
```

Comment:
Why is this needed now but it wasn't before?

Reply:
It used to be added as part of class setup. But this feature is really only relevant for this test so I figured it makes sense to pull it locally.